examples/bzlmod/py_proto_library/example.com/proto/BUILD.bazel

```diff
@@ -1,5 +1,5 @@
-load("@rules_proto//proto:defs.bzl", "proto_library")
-load("@rules_python//python:proto.bzl", "py_proto_library")
+load("@com_google_protobuf//bazel:proto_library.bzl", "proto_library")
+load("@com_google_protobuf//bazel:py_proto_library.bzl", "py_proto_library")
```

Comment:
The python rules are in the protobuf repo directly, not in a python-specific repo? I thought I remember seeing a BCR PR where they were adding various proto_{java,python,etc} repos so e.g. java wasn't pulled in when only python was used.

Reply:
> The python rules are in the protobuf repo directly, not in a python-specific repo?
Yes.

> BCR PR where they were adding various proto_{java,python,etc} repos 

Hm, do you mean `rules_proto_grpc_*`? I imagine those support gRPC, but I wasn't part of that effort.